content/rule_book/quiz_events.md

```diff
@@ -1,117 +1,113 @@
 
-Mispronounced names are still recognizable as the answer.
+**Quiz-Out**
+: 4 correct questions by the same quizzer in the same quiz
 
-Only the first response of the quizzer will be considered, however, if the response by the quizzer is in context and is not incorrect, the quizzer will be allowed their full 30 seconds to satisfy all requirements of a correct response.
+A quizzed-out quizzer may only jump on Bonus questions.
 
 #### Example
 
-INT Who has been making it grow? A. God. If the quizzer said "Apollos" they have given incorrect information (even though the name "Apollos" is in context) and would be called incorrect. If they had said "I planted the seed", "Apollos watered it", they have not given any incorrect information, are in context, and will be given the 30 seconds to supply the correct answer required.
+Consider 1 Corinthians 3:6:
+
+> I planted the seed, Apollos watered it, but God has been making it grow.
+
+Suppose an INT question was written:
+
+> Who has been making it grow?
+
+The correct answer is "God".
+
+If a quizzer answered "Apollos", they have given incorrect information (even though the name "Apollos" is in context) and would be immediately called incorrect. If they had said "I planted the seed, Apollos watered it", they have not given any incorrect information, are in context, and will be given the full question answer duration period to supply the correct answer required.
 
 ### Incorrect
 
 The quizzer's response is incorrect when:
 
-- An Incorrect answer is also known as an error.
-- Incorrect information has been given.
-- The information given within the time limit is incomplete.
-- It does not fulfill all requirements specific to the Question Type.
-- Three Incorrect answers by a quizzer eliminate that quizzer from the quiz. The quizzer must leave the platform immediately.
-    - Incorrect answers on bonus questions do not count towards this total.
+- Incorrect information has been given
+- The information given within the time limit is incomplete
+- The response does not fulfill all requirements specific to the question type
 
-#### Example
+**Error**
+: An incorrect answer is also known as an error
 
-Incorrect Information, e.g. If the wrong person (i.e. Father, Son, Holy Spirit) in the deity (God) is given.
+**Err-Out**
+: 3 errors by the same quizzer in the same quiz
 
-All requirements are not fulfilled, Any word is omitted or added in a Finish-The-Verse, Finish-This, or Quote Question or the wrong verse is begun.
+An err-out eliminates that quizzer from the quiz. The quizzer must leave the platform immediately.
 
 ## Challenges
 
-Challenges exist so quizzers can contest rulings made by quiz officials.
-
-The team captain or co-captain may challenge if:
+Challenges exist so quizzers can contest rulings made by quiz officials. The team captain or co-captain may challenge if:
 
 - There is reason to believe the quiz officials have applied the rule book incorrectly
```

Comment:
Technically "there is reason to believe" applies to each requirement. It seems weird to state it 1/3 times. Mispronunciations and invalids are not 100% objective black and white.

Reply:
Agreed. db92815